prboom2/src/m_cheat.c

```diff
@@ -194,12 +195,38 @@ cheatseq_t cheat[] = {
   // Enable/disable shorttics in-game
   CHEAT("tntshort",   NULL,               cht_never, cheat_shorttics, 0),
 
+  // subscribe to dest alsa port manually
```

Comment:
Sorry, but this is unacceptable. A music backend that requires typing a cryptic cheat to work at all will go down in history as the one that nobody used, even on Linux, even with ALSA midi setup.

Reply:
It's for debugging purposes, it's not meant to be used by the end user. I can remove it if you prefer. I still have to find a way to list possible destination MIDI ports in ALSA, though.